### DISCUSSION TITLE
Python tests on Mac

### DISCUSSION THREAD

**Discussion by RPirie96** [rdkit/rdkit#7830] [Q&A]
Currently writing some docs on how to contribute to RDKit. In fact-checking that the things included are correct I've tried a build for python-dev following [Greg's instructions](https://greglandrum.github.io/rdkit-blog/posts/2020-03-30-setting-up-an-environment.html) on my macbook (M2, MacOS Sonoma 14.5). 

When I run the tests nothing gets returned, and I can't find the test_list.py file anywhere in the repo. Has this been updated, do I need to manually write the test_list.py file or generate it somehow, is this a Mac issue or am I missing something obvious?

**Comment by bp-kelley** [comment]
test_list.py was removed a while ago, I think the docs need to be updated.  We've switched over to using pytest which is way easier IMO.

here is the commit with the change and the rationale.

https://github.com/rdkit/rdkit/pull/5916

**Reply by bp-kelley** [reply]
I'll add an issue to remove the traces of test_list.py
